### DISCUSSION TITLE
Are there tags for the server aka museum?

### DISCUSSION THREAD

**Discussion by kastl-ars** [ente-io/ente#3732] [Q&A]
Hi all,

I see that the `server` component (aka museum) is being released as a package, but apparently it does not have tags/releases? Or did I overlook them in the long list of different tags for `photos`, `auth`, `fdroid` etc.?

Kind Regards,
Johannes

P.S.: To get to know Ente, I wanted to have a look at packaging it for openSUSE. The CLI works, museum would be the next thing...

**Comment by mnvr** [comment]
You're right, there are no repo tags (the images themselves are tagged with the git commit). Server images are published infrequently, but when they are published, the current workflow is to make them using the the git commit that is running on the server (not the current main), using something like

```sh
gh workflow run server-publish.yml -F commit=`curl -s 'https://api.ente.io/ping' | jq -r .id`
```

* The image is made using this workflow - https://github.com/ente-io/ente/blob/main/.github/workflows/server-publish.yml
* The image is tagged with the git commit
* You can see an example run here - https://github.com/ente-io/ente/actions/runs/11162355553
